gnat2goto/driver/tree_walk.adb

```diff
@@ -1575,25 +1788,12 @@ package body Tree_Walk is
             Sym_Nondet   : constant Irep :=
               Fresh_Var_Symbol_Expr (Type_Irep, Func_Str);
             SE_Call_Expr : constant Irep :=
-              New_Irep (I_Side_Effect_Expr_Function_Call);
+              Make_Assume_Expr (N, Make_Range_Expression (Sym_Nondet,
```

Comment:
:pick: It might make sense to put the definition of `Make_Assume_Expr` and `Make_Assert_Call` in a separate commit, although I'd see that as pretty optional